docs/src/index.md

```diff
@@ -58,29 +58,7 @@ diagrams, Petri nets, and the like.
 
 ### What is a GAT?
 
-Generalized Algebraic Theories (GATs) are the backbone of Catlab so let's expand a bit on GATs and how they fit into the bigger picture of algebra.
```

Comment:
Same question about this content. Also, the link that it is replaced with doesn't (yet?) work.

Reply:
Yeah, I'm not sure what's going on with GATlab docs.